tests/integration/solidity_contracts/PlainOpcodes/test_plain_opcodes.py

```diff
@@ -7,6 +7,41 @@
 @pytest.mark.asyncio
 @pytest.mark.IntegrationTestContract
 class TestPlainOpcodes:
+    class TestBlockhash:
+        async def test_should_return_blockhash(
+            self,
+            deploy_solidity_contract: Callable,
+            addresses,
+            blockhashes,
+        ):
+            counter = await deploy_solidity_contract(
```

Comment:
why do we deploy Counter here?

Reply:
Ah I forgot to remove them when I was copy pasting from another test case. Let me update them for `BLOCKHASH` and `ADDRESS` as they're not needed 🤙 we can use a random address for the constructor for `PlainOpcodes.sol`